### PR TITLE
Topic Page - All Translations

### PR DESCRIPTION
**Pull request by RetiredBlob**
Resolves #NUMBER

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/TopicPage/Pagination/index.jsx [hunk 1]
```diff
@@ -191,29 +191,44 @@ const renderBlock = ({
...
 const Pagination = ({ activePage, pageCount }) => {
-  const { service } = useContext(ServiceContext);
+  const { service, translations } = useContext(ServiceContext);
   const blocks = buildBlocks(activePage, pageCount);
   if (!blocks) return null;
 
+  const { pageXOfY, previousPage, nextPage } = getTranslations(translations);
+  const summaryText = pageXOfY
+    .replace('{x}', `<b>${activePage}</b>`)
+    .replace('{y}', `<b>${pageCount}</b>`);
```
Comment (RetiredBlob) [inline src/app/pages/TopicPage/Pagination/index.jsx]: Seeking suggestions on a better way of doing this - can't think of anything that doesn't require either `dangerouslySetInnerHTML`, a mountain of code or a dependency

File: src/app/pages/TopicPage/Pagination/index.jsx [hunk 2]
```diff
@@ -191,29 +191,44 @@ const renderBlock = ({
...
+      )}
       <TextSummary
         service={service}
         data-testid="topic-pagination-summary"
         // eslint-disable-next-line jsx-a11y/aria-role
         role="text"
-      >
-        Page <b>{activePage}</b> of <b>{pageCount}</b>
-      </TextSummary>
+        dangerouslySetInnerHTML={{ __html: summaryText }}
```
Comment (pvaliani) [inline src/app/pages/TopicPage/Pagination/index.jsx]: Just curious why `dangerouslySetInnerHTML` for the summaryText is set within the `Nav` here?

Comment (RetiredBlob) [reply src/app/pages/TopicPage/Pagination/index.jsx]: This is just the github diff being confusing 😕  it's actually on the `TextSummary` component

The split diff view makes it clearer what changed
Screen Shot 2022-04-19 at 09 31 02

**Review by pvaliani — APPROVED** [review]

File: src/app/lib/config/services/kyrgyz.js [hunk 3]
```diff
@@ -59,6 +59,11 @@ export const service = {
     showAdPlaceholder: true,
     showRelatedTopics: true,
     translations: {
+      pagination: {
+        previousPage: 'Артка',
+        nextPage: 'Эмики барак',
+        pageXOfY: 'бет {x} of {y}',
```
Comment (rebeccamcginn) [inline src/app/lib/config/services/kyrgyz.js:65]: "of" is Latin here 🤔

File: src/app/lib/config/services/russian.js [hunk 4]
```diff
@@ -122,6 +122,11 @@ const headerFooterTranslations = {
 
 // Translations used in the main element of the page
 export const mainTranslations = {
+  pagination: {
+    previousPage: 'НАЗАД',
+    nextPage: 'ВПЕРЕД',
+    pageXOfY: 'Page {x} из {y}',
```
Comment (rebeccamcginn) [inline src/app/lib/config/services/russian.js:128]: I don't think they mean for "Page" to be here. Just `{x} из {y}`

File: src/app/lib/config/services/uzbek.js [hunk 5]
```diff
@@ -59,6 +59,11 @@ export const service = {
     showAdPlaceholder: true,
     showRelatedTopics: true,
     translations: {
+      pagination: {
+        previousPage: 'Олдингиси',
+        nextPage: 'Кейингиси',
+        pageXOfY: 'саҳифа {x} ...нинг {y}',
```
Comment (rebeccamcginn) [inline src/app/lib/config/services/uzbek.js:65]: This one is odd

**Review by rebeccamcginn — APPROVED** [review]

**Comment by greenc05** [comment]
@ryanmccombe did we also mean to add the translation for 'Page' and 'page'? I believe the former is required for the aria-label for the navigation element and the later for the aria-current? Or are these already added/awaiting the translation text from editorial?

**Pull request merged**